src/content/api-reference/extended-types/index.d.ts

```diff
@@ -1,4 +1,4 @@
-import type { PropType } from "@/types";
+import type { PropType } from "@/types/index.js";
```

Comment:
This file also seems like it shouldn't be here either, though, it wasn't added in this PR.

Reply:
I think you're right. I went ahead and deleted index.d.ts.